lambda_src/lambda_function.py

```diff
@@ -209,7 +209,7 @@ def sync_flow(event: Any, context: Any = None) -> Optional[ResponseType]:
         end_time = timer()
         if response and (end_time - start_time) > 20:
```

Comment:
Could you put this into a constant with a descriptive name. We should generally try to avoid stray numbers like this as it make the code less readable.

Reply:
Updated it 👍